docs/Usage.md

```diff
@@ -52,7 +52,7 @@ We need to add the IPAM provider to your clusterctl config file `~/.cluster-api/
 ```yaml
 providers:
   - name: in-cluster
-    url: https://github.com/kubernetes-sigs/cluster-api-ipam-provider-in-cluster/releases/download/v0.1.0-alpha.3/ipam-components.yaml
+    url: https://github.com/kubernetes-sigs/cluster-api-ipam-provider-in-cluster/releases/v0.1.0-alpha.3/ipam-components.yaml
```

Comment:
I don't know how that's meant to work, the proposed url is a 404 for me.

Reply:
Dont test the url. 

Test clusterctl

Reply:
release not found for version download, plea

Reply:
**release not found for version download**